scripts/assets/extension-data.yaml

```diff
@@ -7,6 +7,7 @@
 # - `containerd_compatible`: defaults to true.
 # - `logo`: Overrides the logo URL.
 
+rancher/application-collection-extension:0.1.0: {}
```

Comment:
I wonder if we should sort this in some manner in the future.

Reply:
We probably should sort it alphabetically. But it is only for maintenance; in the app the extensions are sorted by title anyways.